quic/s2n-quic-events/events/common.rs

```diff
@@ -826,9 +826,11 @@ enum EcnState {
 }
 
 /// Events tracking the progress of handshake status
-enum HandshakeStatus {
+enum HandshakeStatus<'a> {
     /// The handshake has completed.
-    Complete,
+    Complete {
+        session: crate::event::TlsSession<'a>,
+    },
```

Comment:
Rather than overloading the HandshakeStatus event, I wonder if it would be better to emit a specific event for the exporter material?

It might simplify the TLS providers that don't provide such functionality, where they just don't emit the event.

Reply:
Yeah, that seems like a good idea.